config/samples/databricks_v1alpha1_djob.yaml

```diff
@@ -1,22 +1,23 @@
 apiVersion: databricks.microsoft.com/v1alpha1
 kind: Djob
 metadata:
-  name: djob-sample
+  name: djob-sample-1
 spec:
   # This spec is directly linked to the JobSettings structure
   # https://docs.databricks.com/api/latest/jobs.html#jobsettings
   new_cluster:
     spark_version: 5.3.x-scala2.11
     node_type_id: Standard_D3_v2
-    num_workers: 3
-  libraries:
-    - jar: 'dbfs:/my-jar.jar'
-    - maven:
-        coordinates: 'org.jsoup:jsoup:1.7.2'
+    num_workers: 10
```

Comment:
Please change back the  num_workers to 3

Reply:
sure good catch

Reply:
done